vj4/ui/templates/partials/contest_status_download_acm.csv

```diff
@@ -0,0 +1,6 @@
+{{ _('Rank') }},{{ _('User') }},{{ _('Solve') }},{{ _('Time') }},{% for pid in tdoc['pids'] %}#{{ loop.index }} {{ pdict[pid]['title'] }},{% endfor %}
```

Comment:
I think there is no need to use template here. Current template is extremely hard to read.